new-dti-website/src/app/initiatives/page.tsx

```diff
@@ -24,11 +24,11 @@ const InitiativeHero = () => (
       <div className="flex flex-col gap-6">
         <h2
           className="font-bold md:text-[40px] xs:text-[24px] md:leading-[48px] 
-          xs:leading-[29px] text-[#877B7B]"
+          xs:leading-[29px] text-heroPrimary"
         >
-          Invigorating <span className="text-[#E4E4E4] italic">growth</span>
+          Invigorating growth
         </h2>
-        <p className="md:text-lg xs:text-sm">
+        <p className="md:text-lg xs:text-sm text-heroSecondary">
```

Comment:
can we vertically center the content on the right side with the left side

just an `align-items: center;` on the main container
![image](https://github.com/user-attachments/assets/b53686b9-7440-43f9-b331-aefb15b5ba86)